### ISSUE TITLE
Fixer conflict: `Squiz.PHP.EmbeddedPhp` vs `WordPress.CodeAnalysis.EmptyStatement`

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#1112]
When running PHPCBF over `src/wp-admin/includes/ms.php` using `WordPress-Core` + `WordPress.CodeAnalysis.EmptyStatement`, I'm running into a fixer conflict.

Minimal code sample to reproduce the issue:
```php
Something
<?php /* translators: My sites label */ ?>
Something else
```

Relevant debug info regarding the loop:
```
        => Fixing file: 1/289 violations remaining [made 46 passes]...  * fixed 1 violations, starting loop 47 *
        PHP_CodeSniffer\Standards\Squiz\Sniffs\PHP\EmbeddedPhpSniff (line 353) replaced token 6635 (
T_OPEN_TAG) "<?php " => "<?php ;"
        => Fixing file: 1/289 violations remaining [made 47 passes]...  * fixed 1 violations, starting loop 48 *
        => Changeset started by WordPress\Sniffs\CodeAnalysis\EmptyStatementSniff (line 80)
                Q: WordPress\Sniffs\CodeAnalysis\EmptyStatementSniff (line 101) replaced token 6636
(T_SEMICOLON) ";" => ""
                A: WordPress\Sniffs\CodeAnalysis\EmptyStatementSniff (line 104) replaced token 6636
(T_SEMICOLON) ";" => ""
        => Changeset ended: 1 changes applied
```

A first look makes me think this is an upstream issue with the `Squiz.PHP.EmbeddedPhp` sniff which tries to add a `;` to the embedded PHP like so: `<?php ;/* translators: My sites label */ ?>`.

**Comment by jrfnl** [comment]
This has been confirmed to be an upstream issue. A fix has been pulled upstream: https://github.com/squizlabs/PHP_CodeSniffer/pull/1633

**Comment by jrfnl** [comment]
The upstream PR has been merged and will be in the PHPCS 3.1.0 release.
